src/components/EntityStatus/EntityStatus.js

```diff
@@ -121,15 +119,13 @@ class EntityStatus extends React.Component {
                     {this.renderLink()}
                 </span>
                 {hasClipboardButton && (
-                    <Button
-                        component="span"
-                        size="s"
+                    <ClipboardButton
```

Comment:
`ClipboardButton` doesn't disappear after copy event until some outside click

<img width="180" alt="Screen Shot 2024-01-15 at 19 56 11" src="https://github.com/ydb-platform/ydb-embedded-ui/assets/67755036/94da6968-8584-4a36-97a2-3e3848c9c1ec">

Reply:
it's a feature, not a bug :)
The button stays focused after being clicked.